docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  web:
+    build: .
```

Comment:
#### Automated PR Comment From Polaris SAST
<!-- 022ebdc9c1d8667c69ad3a396a48408c -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Polaris SAST Issue - Improper Access Control
_Low CWE-284_
The Docker container is not configured to drop all default capabilities and then add only required one. Thus the container may be granted excessive privileges which violate the least privilege principle.



### How to fix
Potential mitigations include the following:
  1. Very carefully manage the setting, management, and handling of privileges. Explicitly manage trust zones in the software.
  2. Compartmentalize the system to have "safe" areas where trust boundaries can be unambiguously drawn. Do not allow sensitive data to go outside of the trust boundary and always be careful when interfacing with a compartment outside of the safe area.

Ensure that appropriate compartmentalization is built into the system design, and the compartmentalization allows for and reinforces privilege separation functionality. Architects and designers should rely on the principle of least privilege to decide the appropriate time to use privileges and the time to drop privileges.